docs/man_bodhi.rst

```diff
@@ -0,0 +1,250 @@
+==================
+bodhi CLI man page
+==================
+
+Synopsis
+========
+
+``bodhi`` [OPTIONS] COMMAND [ARGS]...
+
+
+Description
+===========
+
+``bodhi`` is the command line interface to bodhi, Fedora's update release management system. It can
+be used to create or modify updates and overrides.
+
+
+Options
+=======
+
+Most of the commands will accept these three options:
+
+``--help``
+
+    Show help text and exit.
+
+``--password <text>``
+
+    A password to authenticate as the user given by ``--user``.
+
+``--staging``
+
+    Use the staging bodhi instance instead of the production instance.
+
+``--user <username>``
+
+    Many commands accept this flag to specify which user's updates should be operated upon.
+
+
+Commands
+========
+
+There are two commands, ``overrides`` and ``updates``. They are described in more detail in their
+own sections below.
+
+``bodhi overrides [options] command <args>``
+
+    Provides commands to aid in management of build overrides. Supports subcommands ``query`` and
+    ``save``, described below.
+
+``bodhi updates [options] command <args>``
+
+    Provides an interface to manage updates. Supports subcommands ``comment``, ``download``,
+    ``new``, ``query``, and ``request``, described below.
+
+
+Overrides
+=========
+
+The ``overrides`` command allows users to manage build overrides.
+
+``bodhi overrides query [options]``
```

Comment:
My guess is that options means things like ``--staging`` and what not, referenced at the top, and args means things specific to the subcommand. I'll do a pass and try to make that consistent.